src/boot/efi/pe.c

```diff
@@ -160,6 +166,45 @@ static void locate_sections(
         }
 }
 
+static UINT32 get_compatibility_entry_address(const struct DosFileHeader *dos, const struct PeFileHeader *pe) {
+        UINTN addr = 0, size = 0;
+        static const CHAR8 *sections[] = { (CHAR8 *) ".compat", NULL };
+
+        locate_sections((struct PeSectionHeader *) ((UINT8 *) dos + section_table_offset(dos, pe)),
```

Comment:
doesn't matter but strictly speaking you cast the const away, that `dos` carries. to be fully correct the two casts would still have it